themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -215,17 +215,22 @@ VuFind.register('itemStatuses', function ItemStatuses() {
     }
   }
   function init(_container) {
-    if (typeof Hunt === 'undefined' || VuFind.isPrinting()) {
-      checkItemStatuses(_container);
-    } else {
-      var container = typeof _container === 'undefined'
-        ? document.body
-        : _container;
-      new Hunt(
-        $(container).find('.ajaxItem').toArray(),
-        { enter: checkItemStatus }
-      );
+    var container = typeof _container === 'undefined'
+      ? document.body
+      : _container;
+
+    if (VuFind.isPrinting()) {
+      checkItemStatuses(container);
+      return;
     }
+    VuFind.observerManager.createIntersectionObserver(
+      'itemStatuses',
+      checkItemStatus
```

Comment:
Might it make sense to add an optional third "initial observation target" parameter to createIntersectionObserver so that everything can be set up in a single step? It might improve the readability of the code.

Reply:
Sounds great. Will do.